app/config.py

```diff
@@ -2,12 +2,14 @@
 
 from pydantic_settings import BaseSettings, SettingsConfigDict
 
+
 class Settings(BaseSettings):
     google_application_credentials: str
 
-    model_config = SettingsConfigDict(env_file=".local_env")
+    model_config = SettingsConfigDict(env_file="../.local_env")
```

Comment:
Rename `.local_env` file to `.env` file following name convention.

Reply:
because venv folder we named .env before, so what should I do to resolve this conflict?